modules/users/module.json

```diff
@@ -26,8 +29,12 @@
     "create": {
       "name": "Create User"
     },
-    "authenticate_token": {
+    "authenticate_token_internal": {
       "name": "Authenticate User Token",
+      "description": "Validate a user token. Throws an error if the token is invalid."
+    },
+    "authenticate_token": {
+      "name": "Public Authenticate User Token",
```

Comment:
default name shoudl be "authenticate user token." change the internal to "authenticate user token (internal)"

we'll clean up public/internal soon